seed/src/lib.rs

```diff
@@ -317,3 +365,28 @@ impl Node {
         Ok(())
     }
 }
+
+/// Guess a user given a peer id.
+async fn guess_user(
```

Comment:
In the words of @xla, "I don't hate this".

Reply:
hahaha